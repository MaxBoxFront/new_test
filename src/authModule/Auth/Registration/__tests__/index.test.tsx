import React from 'react';
import * as redux from "react-redux";

jest.mock("react-redux", () => ({
	useSelector: jest.fn(fn => fn()),
}));

describe('The Registration component', () => {
	it('renders correctly', () => {

		const spy = jest.spyOn(redux, 'useSelector')
		spy.mockReturnValue({ username:'' })
	});
});