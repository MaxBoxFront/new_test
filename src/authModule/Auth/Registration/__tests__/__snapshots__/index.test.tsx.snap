// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Registration component renders correctly 1`] = `
<div
  className="registration"
>
  <div
    className="content"
  >
    <h2>
      Create account
    </h2>
    <div
      className="input__wrapper input"
    >
      <input
        className="input input--fluid"
        id="username"
        onChange={[Function]}
        placeholder="username"
        type="text"
        value=""
      />
    </div>
    <div
      className="input__wrapper input"
    >
      <input
        className="input input--fluid"
        id="email"
        onChange={[Function]}
        placeholder="Email"
        type="text"
        value=""
      />
    </div>
    <div
      className="input__wrapper input"
    >
      <input
        className="input input--fluid"
        id="password"
        onChange={[Function]}
        placeholder="Password"
        type="text"
        value=""
      />
    </div>
    <button
      className="button button--fluid button"
      onClick={[Function]}
      type="button"
    >
      Create account
    </button>
  </div>
</div>
`;
